driver/src/main/scala/com/stratio/sparkta/driver/SparktaClusterJob.scala

```diff
@@ -89,7 +68,7 @@ object SparktaClusterJob extends SparktaSerializer {
 
         val streamingContextService = new StreamingContextService(Some(policyStatusActor))
         val ssc = streamingContextService.clusterStreamingContext(
-          policy, pluginsClasspathFiles, Map("spark.app.name" -> s"${policy.name}")).get
+          policy, Seq(), Map("spark.app.name" -> s"${policy.name}")).get
```

Comment:
If always there is an empy Seq() in the sign of the method remove it from the function.